CMakeLists.txt

```diff
@@ -1,13 +1,19 @@
 cmake_minimum_required(VERSION 3.10)
-project(vikunja)
+project(vikunja
+  VERSION      0.1.0
+  DESCRIPTION  "Primitives for Alpaka"
+  HOMEPAGE_URL "https://github.com/ComputationalRadiationPhysics/vikunja"
+  LANGUAGES    CXX)
 
 set(CMAKE_CXX_STANDARD 14)
```

Comment:
That suggests C++14 for all subprojects. To enforce it, you also have to set `CMAKE_CXX_STANDARD_REQUIRED`.

Recently, I have also become a fan of `target_compile_features(${PROJECT_NAME} INTERFACE cxx_std_17)` for interface libraries, because you just state that the library needs C++17 and any project downstream can request something higher.